k8s/cassandra/apptest/tester/tests/basic-suite.yaml

```diff
@@ -38,11 +38,37 @@ actions:
 - name: Execute CQL query
```

Comment:
Can we name steps in a question form (here and below)?

```suggestion
- name: Can execute CQL query
```

Reply:
I do not like question style, please do not for it here on me.